pkg/controller/disruption/disruption.go

```diff
@@ -596,13 +588,13 @@ func (dc *DisruptionController) getExpectedScale(pdb *policy.PodDisruptionBudget
 	for _, pod := range pods {
 		controllerCount := 0
 		for _, finder := range dc.finders() {
-			var controllers []controllerAndScale
-			controllers, err = finder(pod)
+			var controllerNScale *controllerAndScale
+			controllerNScale, err = finder(pod)
```

Comment:
Ah I see. I didn't notice this function is using named return values.

Technically since this would be a multi-variable short declaration and `controllerNScale` would be a new variable, `err` would not be shadowed but rather reused. However, I agree it's better to leave it as is rather than relying on that obscure rule.